src/silo/query_engine/filter_expressions/insertion_contains.cpp

```diff
@@ -55,28 +56,14 @@ std::unique_ptr<silo::query_engine::operators::Operator> InsertionContains<Symbo
       return std::make_unique<operators::Empty>(database_partition.sequence_count);
    }
 
-   std::string validated_sequence_name;
-   if (sequence_name.has_value()) {
-      validated_sequence_name = sequence_name.value();
-   } else {
-      CHECK_SILO_QUERY(
-         database.getDefaultSequenceName<SymbolType>().has_value(),
-         "The database has no default " + std::string(SymbolType::SYMBOL_NAME_LOWER_CASE) +
-            " sequence name"
-      )
-      // NOLINTNEXTLINE(bugprone-unchecked-optional-access) -- the previous statement checks it
-      validated_sequence_name = *database.getDefaultSequenceName<SymbolType>();
-   }
+   std::string valid_sequence_name =
+      getValidSequenceNameOrDefault<SymbolType>(sequence_name, database);
+
    const std::map<std::string, SequenceStorePartition<SymbolType>&>& sequence_stores =
       database_partition.getSequenceStores<SymbolType>();
-   CHECK_SILO_QUERY(
-      sequence_stores.contains(validated_sequence_name),
```

Comment:
Is there a specific reason you removed this check?

Reply:
A misconfigured "default sequence" would lead to an error here. Unless we can circumvent that elsewhere

Reply:
I will enable the check again, and add a ticket to fix this when we have time.

Reply:
Ticket: #475